lib/Service/Appointments/EventConflictFilter.php

```diff
@@ -63,28 +65,30 @@ public function filter(AppointmentConfig $config, array $slots): array {
 		$query->addType('VEVENT');
 		$preparationDuration = DateInterval::createFromDateString($config->getPreparationDuration() . ' seconds');
 		$followUpDuration = DateInterval::createFromDateString($config->getFollowupDuration() . ' seconds');
-		$this->logger->debug('Appointment config ' . $config->getToken() . ' is looking with preparation duration {preparation} and follow up duration {followup} in calendar {calendarUri}', [
-			'preparation' => $preparationDuration->s,
-			'followup' => $followUpDuration->s,
-			'calendarUri' => $config->getTargetCalendarUri()
-		]);
-		return array_filter($slots, function (Interval $slot) use ($followUpDuration, $preparationDuration, $query): bool {
+		return array_filter($slots, function (Interval $slot) use ($followUpDuration, $preparationDuration, $query, $config): bool {
 			$query->setTimerangeStart($slot->getStartAsObject()->sub($preparationDuration));
 			$query->setTimerangeEnd($slot->getEndAsObject()->add($followUpDuration));
 
 			$events = $this->calendarManager->searchForPrincipal($query);
-			$objects = array_filter($events, function (array $event) {
+
+			$uids = '';
```

Comment:
you can do better than this :wink: 

revert to the previous filtered array. `$object` contains all the conflict events. Array map them to their UID/id (or URI?) and implode the result.

Mutations are not our friends. Let's avoid them :)

Reply:
Yes sorry I'm super tired 😅